mongodbatlas/data_source_mongodbatlas_advanced_clusters.go

```diff
@@ -221,7 +223,11 @@ func flattenAdvancedClusters(ctx context.Context, d *schema.ResourceData, conn *
 	results := make([]map[string]interface{}, 0)
 
 	for i := range clusters {
+		processArgs, _, err := conn.Clusters.GetProcessArgs(ctx, clusters[i].GroupID, clusters[i].Name)
+		log.Printf("[WARN] Error setting `advanced_configuration` for the cluster(%s): %s", clusters[i].ID, err)
+
```

Comment:
[q] Shouldn’t we check that `err != nil` before printing the log message?

Reply:
You're right, will add the validation, thank you